src/js/Options.js

```diff
@@ -6,18 +6,13 @@ const options = {
 
 };
 
-let admOptionValue = document.getElementById("local").value;
-{
-    if (nameSelect) {
-        admOptionValue = document.getElementById("local").value;
-        if (admOptionValue == nameSelect.value) {
-            document.getElementById("admDivCheck").style.display = "block";
-        }
-        else {
-            document.getElementById("admDivCheck").style.display = "none";
-        }
+let typeSelect = document.getElementById("type")
+typeSelect.addEventListener("change", function () {
```

Comment:
First, the `<select>` is found with `getElementById`. To add an event handler, `addEventListener` is used. The first argument `"change"` is the event type (onchange), and the second argument is a function. This function is called whenever the element is changed. 

This statement is a function call: `typeSelect.addEventListener("change", ......);`, and the handler is a new function `function () { ...... }`. The function does not need a name.